internal/cluster/cluster.go

```diff
@@ -775,11 +780,28 @@ func (m *Manager) UpdateFinalizingProgress(ctx context.Context, db *gorm.DB, clu
 			common.ProgressWeightInstallingStage*float64(cluster.Progress.InstallingStagePercentage))
 	}
 
+	if cluster.Progress != nil && totalPercentage < cluster.Progress.TotalPercentage {
+		log.Debugf(
+			"cluster %s: skipping progress_total_percentage update.The new progress_total_percentage is: %d, which is lower then the current cluster progress_total_percentage: %d",
+			clusterID.String(), totalPercentage, cluster.Progress.TotalPercentage)
+		return nil
+	}
+
 	updates := map[string]interface{}{
 		"progress_finalizing_stage_percentage": finalizingStagePercentage,
 		"progress_total_percentage":            totalPercentage,
 	}
 
+	// If progress has reached 95%, cluster monitoring should be triggered.
+	// Running cluster monitoring will allow a faster update of the cluster status and indicate that the installation has been completed,
+	// as soon as Kubeconfig is available for that cluster.
+	if totalPercentage == common.TotalProgressWithoutKubeconfig {
+		log.Debugf(
+			"cluster %s: progress_total_percentage reached %d, setting trigger_monitor_timestamp to trigger cluster monitoring",
+			clusterID.String(), common.TotalProgressWithoutKubeconfig)
+		updates["trigger_monitor_timestamp"] = time.Now()
+	}
```

Comment:
1. If an optional operator fails, we'll never reach this
2. Won't cluster monitoring happen anyway once the installation is complete?

Reply:
[These events](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_assisted-service/3519/pull-ci-openshift-assisted-service-master-e2e-metal-assisted/1509187606733131776/artifacts/e2e-metal-assisted/assisted-baremetal-gather/artifacts/2022-03-30_16:01:22_e2f15a45-6075-488d-8a2b-9098a46bb13f/events.html) show that without the update to `trigger_monitor_timestamp`, timing remained the same.

Reply:
Removed